jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -442,7 +447,12 @@ def create_bg(self, add_data=False):
         return bg_spec
 
     def vue_create_bg(self, *args):
-        _ = self.create_bg(add_data=True)
+        try:
+            _ = self.create_bg(add_data=True)
+        except Exception as e:
+            msg = (f"Specreduce background failed with the following error: {str(e)}")
```

Comment:
Including ErrorClass is more useful here.
```suggestion
            msg = (f"Specreduce background failed with the following error: {repr(e)}")
```